.gitignore

```diff
@@ -51,3 +51,8 @@ build-iPhoneSimulator/
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+
+# Specific to Rubymine
+.rakeTasks
+.idea
```

Comment:
```suggestion
```

IMO, let's avoid editor-specific config in this file. This kind of information is probably better off in a local, global gitignore.